syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -631,22 +633,38 @@ def float_prec_(self):
         if hasattr(tensor, "child"):
             self.child = tensor.child
         elif self._is_parameter():
+            self.is_wrapper = False
             self.data = tensor
+            self.data.is_wrapper = False
         else:
             del self.child
             self.set_(tensor)
+            self.is_wrapper = False
         return self
 
     float_precision_ = float_prec_
 
-    def fix_prec(self, *args, storage="auto", field_type="int100", **kwargs):
+    def fix_prec(self, *args, storage="auto", field_type="int100", no_wrap: bool = False, **kwargs):
+        """
+        Convert a tensor or syft tensor to fixed precision
+
+        Args:
+            args (tuple): args to transmit to the fixed precision tensor
```

Comment:
nit could you denote these as *args and **kwargs in the docstring so it is clear they are of arbitrary length